app/static/src/app/components/options/RunOptions.vue

```diff
@@ -59,6 +61,26 @@ export default defineComponent({
         const numberOfReplicates = computed(() => store.state.run.numberOfReplicates);
         const isStochasticApp = computed(() => store.state.appType === AppType.Stochastic);
 
+        const maxReplicatesDisplay = computed(() => {
+            return (store.state.config as StochasticConfig)?.maxReplicatesDisplay || 50;
+        });
+        const maxReplicatesRun = computed(() => {
+            return (store.state.config as StochasticConfig)?.maxReplicatesRun || 1000;
+        });
+        const maxAllowedArray = computed(() => {
+            return [
+                {
+                    number: maxReplicatesDisplay.value,
+                    message: "Individual traces will not be shown past this point",
```

Comment:
```suggestion
                    message: `Individual traces will not be shown for values greater than ${maxReplicatesDisplay}`,
```

Reply:
I think we need to tell users what the cut off point actually is.

Reply:
done